### PR TITLE
Optional questions

### PR DESCRIPTION
**Pull request by JuliaJMP**
If the question is option you can skip the question

### CODE REVIEW

**Comment by Sermodi** [comment]
:+1:

**Comment by carloscharx** [comment]
:100:

File: app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java [hunk 1]
```diff
@@ -58,6 +61,9 @@ public void onResponse(LoginModel response) {
             mActivity.getApplication();
             Intent intent = new Intent(mContext, MainActivity.class);
             intent.putExtra("token", response.getToken());
+            if (mCheckBox.isChecked()){
+                intent.putExtra(getString(R.string.remember_checked_intent), true);
```
Comment (jrodgut-sg) [inline app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java]: it is not needed to place this in the Android Strings, a java variable would be enough

File: app/src/main/java/es/academy/solidgear/surveyx/ui/activities/MainActivity.java [hunk 2]
```diff
@@ -21,14 +24,22 @@ protected void onResume() {
 
         Bundle extras = getIntent().getExtras();
         String token = null;
+        boolean isRememberChecked = false;
         if (extras != null) {
             token = extras.getString("token", null); /** TODO: Read token when it will be stored **/
+            isRememberChecked = extras.getBoolean(getString(R.string.remember_checked_intent), false);
```
Comment (jrodgut-sg) [inline app/src/main/java/es/academy/solidgear/surveyx/ui/activities/MainActivity.java]: it is not needed to place this in the Android Strings, a java variable would be enough

**Pull request merged**